ob_intern/Controller/Short Video Post/ShortVideoPostViewController.swift

```diff
@@ -22,21 +23,16 @@ class ShortVideoPostViewController: UIViewController {
     
     //MARK: - IBOutlet
     @IBOutlet weak var tableView: UITableView!
+    @IBOutlet weak var buttonBack: UIButton!
     
     //MARK: - Parameters
     private var viewModel: ShortVideoPostViewModel?
-    var caption: [String] = ["Lorem ipsum dolor sit amet, consetetur sadipscing elitr, sed diam nonumy eirmod tempor invidunt ut labore et dolore magna aliquyam erat, sed diam voluptua. At vero eos et accusam et justo duo doloreeeeeeeeeeeeeeeee…", "ajdnklanfk kajndskcnkadsnc kjandckjandkjc nakjdcnkajndc naksdjcna  ncaskdjcna", ""]
-    var mockImageUrls: [String] = ["https://images3.alphacoders.com/110/1108129.jpg",
-                                   "https://wallpaperaccess.com/full/6193236.jpg",
-                                   "https://imgix.bustle.com/uploads/image/2022/2/11/c277a32f-c52c-4d7a-98ea-1a0bbec3cf2d-baby-yoda-use-the-force.jpg?w=1200&h=630&fit=crop&crop=focalpoint&fm=jpg&fp-x=0.4813&fp-y=0.3059",
-                                   "https://i.pinimg.com/originals/4e/52/2d/4e522df5de3a6903cf2272572eb471aa.jpg"]
-    var hashtag: [Bool] = [true, false, true]
-    
     
     //MARK: - Lifecycle
     override func viewDidLoad() {
         super.viewDidLoad()
         self.setupView()
+        self.viewModel?.getVideoPost()
         tableView.reloadData()
```

Comment:
ถ้าจะใช้ self นำหน้า ซอฟต์ทำอันนี้ด้วยจะได้ consistency กันเนาะ 
```suggestion
        self.tableView.reloadData()
```